### PR TITLE
Add github issue and PR templates

### PR DESCRIPTION
**Pull request by ryelle**
This adds a simple PR template, you can [see it here.](https://raw.githubusercontent.com/WordPress/wordcamp.org/add/github-templates/.github/pull_request_template.md) We can add any extra notes or steps, but this seems like a good start.

There's also a set of issue templates, which will give people the ability to pick which template to use, like [woocommerce](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/new/choose) and [gutenberg](https://github.com/wordpress/gutenberg/issues/new/choose). I've added "Bug Report", "Feature Request" and "Support Question" - with the last really linking people off to other places to get help.

Finally, I updated the main readme to let folks know that we address new issues/PRs once a month, and to add a note about the monthly meta bug scrubs.

Outside of this PR, I've also updated [the descriptions of labels](https://github.com/WordPress/wordcamp.org/labels) with some notes we made in team chats– specifically:

- [Status] Needs Decision: Needs full team input, decision should be reached within a week
- [Status] Blocked: Progress is blocked, and needs team input to move forward
- **NEW** [Status] On Hold: External blockers, or a de-prioritized project
- **NEW** [Status] Ready to Merge _I added this one a while ago, it's helpful for taking approved PRs out of the "Needs Review" state_

The On Hold status is to differentiate "blocked by team input/work" from "blocked by an external issue", and to give PRs that aren't a priority right now a place to live.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: `MAYBE` - I'd remove this section, since it feels like a lot of work to fill it in w/ specific version numbers etc, and the vast majority of issues can be reproduced without it. It feels like it adds friction to the process without much benefit. We can always follow up and ask for this info if we can't reproduce something in the browsers we have installed locally.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: `MAYBE` I'd move this to the top, and maybe make it an HTML comment so people don't leave it in the report through laziness.

Maybe change the `additional context` heading to `Security Issues` or something, so it's more descriptive.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: This is a really good idea. People often report stuff w/out telling us which camp they're working on. 

`MAYBE` - I wonder if it'd be better to ask for a specific URL instead, since they might not make it clear which specific screen they're referring to.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/bug-report.md:13]: When I [preview](https://github.com/WordPress/wordcamp.org/blob/3e573def4a32f2b974eaf39475eaab6ba0d49e25/.github/ISSUE_TEMPLATE/bug-report.md) all of these section headers are missing the blank line that they have here.

`SHOULD` - It seems like we should use `###` instead of `**` so that they're actual headings.

This applies to the other files as well.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/bug-report.md]: This heading has a `:` but the others don't. I think it should be consistent. 

`SHOULD` - I'd prefer to remove the `:` since IIRC most English style guides recommend against punctuation in headings, but don't feel strongly either way.

File: .github/ISSUE_TEMPLATE/support-request.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/support-request.md:7]:
```suggestion
If you have a question about using your site, or contributing to the platform, please email support@wordcamp.org or join the `#meta-wordcamp` channel in [the official WordPress Slack workspace](https://make.wordpress.org/chat/).
```

Same applies to the URLs below.

File: .github/pull_request_template.md
Comment (iandunn) [inline .github/pull_request_template.md:18]: `MAYBE` - I think committers will already know to do this, and everyone else won't be able to add labels, so it'd probably be best to remove this.

File: .github/ISSUE_TEMPLATE/feature-request.md
Comment (iandunn) [inline .github/ISSUE_TEMPLATE/feature-request.md]: `MUST` - Feature requests need to be approved by the Community Team before we can accept them, so I think we should reflect that here.

`MAYBE` - We could just remove all of these sections and just ask people to follow this instead:

https://make.wordpress.org/community/feature-requests-for-community-sites/

...or, maybe we should add a warning (similar to security in the bug report template) to follow that process first, and then once it's been approved, then they can open an issue here, and the template should ask them to link to the p2 post where it was approved. 

If we do that, there shouldn't be a need to describe to problem or alternatives, since those should already be worked through in the p2 post. We'd only need a description of the solution that was agreed upon on the p2.

**Review by iandunn — DISMISSED** [review]
Overall it's really good, kudos!

IMO the feature request approval is a blocker, but everything else is subjective.

File: .github/ISSUE_TEMPLATE/support-request.md
Comment (ryelle) [reply .github/ISSUE_TEMPLATE/support-request.md:7]: Ideally they'd never see the content post-markdown-conversion though, because this shows up in the issue text editor and the goal is to prevent people from submitting these as issues. So adding `[]()` just adds visual noise to the editor.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (ryelle) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: I'd rather leave it- I've had a lot of issues created where I follow up to ask for this info and the person never replies.

browser/OS is also more important in JS issues, so as we're building more JS-focused things, this will be more useful.

File: .github/pull_request_template.md
Comment (ryelle) [reply .github/pull_request_template.md:18]: It's a good reminder, I'd rather leave it.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (coreymckrill) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I agree, this is awesome, and also a specific URL would be more helpful.

File: .github/ISSUE_TEMPLATE/feature-request.md
Comment (coreymckrill) [reply .github/ISSUE_TEMPLATE/feature-request.md]: > the template should ask them to link to the p2 post where it was approved

+1 for this.

File: .github/pull_request_template.md
Comment (coreymckrill) [reply .github/pull_request_template.md:18]: Yeah, I think these templates should be used by committers also, so I agree it's a good reminder.

File: .github/ISSUE_TEMPLATE/support-request.md
Comment (iandunn) [reply .github/ISSUE_TEMPLATE/support-request.md:7]: Ah, that's a great point 👍

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (iandunn) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: In cases when you can't reproduce it locally in the current version of the major browsers, I would assume that the most likely cause is a browser extension. Is that true in your experience, or did most of the issues that came down to differences in supported versions of the browsers?

It seems like if we can't reproduce it, and the reporter doesn't care enough about it to respond, then we could just close it as `maybelater` and let another reporter re-open it if someone ever runs into it again.

Comment (ryelle) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Having them report the browser short-circuits the need to trial-and-error tests among all browsers (start in Chrome, then try again in FF, maybe now try Safari, then boot the VM to try Windows + Chrome, Firefox, IE…).

Adding a note about trying with browser extensions disabled could be a good idea though.

Comment (iandunn) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: To me the amount of friction for the user and the dev feels about the same in those cases, but we'd be adding that friction for users every time in order to save us from that friction in a small percent of cases.

So, personally, it still feels like it hurts them more than it helps us.

**Comment by ryelle** [comment]
I've (finally) addressed the comments on the templates – @coreymckrill do you want to give this another 👀 before I merge it? We can always tweak the templates once merged if we need to.

**Review by coreymckrill — APPROVED** [review]
👍

**Pull request merged**
